cg/cli/workflow/balsamic/options.py

```diff
@@ -31,3 +39,9 @@
     help="Job priority in SLURM. Will be set automatically according to priority in ClinicalDB, \
```

Comment:
help text seems incorrect

Reply:
Yes, I corrected it: [f374510](https://github.com/Clinical-Genomics/cg/pull/1477/commits/f37451062ed33427c9106372b8f145b6c83b634f)